engine/hatchery/vsphere/vsphere.go

```diff
@@ -32,6 +32,14 @@ func (h *HatcheryVSphere) ApplyConfiguration(cfg interface{}) error {
 		return fmt.Errorf("Invalid configuration")
 	}
 
+	// h.Client = cdsclient.NewService(h.Config.API.HTTP.URL, 60*time.Second)
```

Comment:
Comment are normal, I avoid update to many hatchery for now, waiting branch from @bnjjj 
They will be useful to have hatchery as a CDS Services